client/components/EntryFormRow/index.js

```diff
@@ -1,20 +1,57 @@
 import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { getCategories } from 'modules/categories';
+import { actions as AppActions } from 'modules/transactions';
 import DataSelector from './DataSelector';
 import styles from './style.scss';
 
+@connect(
```

Comment:
I brought the same questions/concerns with Grgur and we ultimately decided on `connect`ing this component. @grgur was this done as you envisioned? I'm open to further suggestions.